modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -45,7 +45,7 @@ has initialized.
 [source,terminal]
 ----
 $ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-instance-group --instance-group-zone=${ZONE_0}
```

Comment:
`--instance-group=${INFRA_ID}-bootstrap-instance-group` should be `--instance-group=${INFRA_ID}-bootstrap-ig`, thanks!